tauri-build/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "tauri-build"
+version = "0.2.0"
+authors = ["Lucas Fernandes Gonçalves Nogueira <lucas@quasar.dev>", "Daniel Thompson-Yvetot <denjell@sfosc.org>"]
```

Comment:
where is @tensor-programming in this line?

Reply:
I don't think I've ever been in any of our toml files haha. Ill put myself in there.